src/Illuminate/Routing/ResourceRegistrar.php

```diff
@@ -41,6 +62,10 @@ public function __construct(Router $router)
      */
     public function register($name, $controller, array $options = [])
     {
+        if (isset($options['wildcards']) && ! isset($this->wildcards)) {
+            $this->wildcards = $options['wildcards'];
```

Comment:
This will break routes that don't subsequently have wildcards set because it will use the old ones due to the ResourceRegistrar being used as singleton in the router class. See my pr which doesn't have that problem #12137